frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -116,16 +116,19 @@
           v-on:overlay-hidden="onPopupDismissed"
           v-if="this.user.loggedIn"
         >
-          <omegaup-arena-runsubmit
-            slot="popup-content"
-            v-bind:preferred-language="problem.preferred_language"
-            v-bind:languages="problem.languages"
-            v-bind:initial-show-form="showFormRunSubmit"
-            v-on:dismiss="onPopupDismissed"
-            v-on:submit-run="
-              (code, selectedLanguage) => onRunSubmitted(code, selectedLanguage)
-            "
-          ></omegaup-arena-runsubmit>
+          <omegaup-overlay-popup slot="popup">
+            <omegaup-arena-runsubmit
+              slot="popup-content"
+              v-bind:preferred-language="problem.preferred_language"
+              v-bind:languages="problem.languages"
+              v-bind:initial-show-form="showFormRunSubmit"
+              v-on:dismiss="onPopupDismissed"
+              v-on:submit-run="
+                (code, selectedLanguage) =>
+                  onRunSubmitted(code, selectedLanguage)
+              "
+            ></omegaup-arena-runsubmit>
+          </omegaup-overlay-popup>
```

Comment:
se puede usar la sintaxis [`v-slot`](https://vuejs.org/v2/guide/components-slots.html#Named-Slots) en vez?

```suggestion
          <template v-slot:popup>
            <omegaup-arena-runsubmit-popup
              v-bind:preferred-language="problem.preferred_language"
              v-bind:languages="problem.languages"
              v-bind:initial-show-form="showFormRunSubmit"
              v-on:dismiss="onPopupDismissed"
              v-on:submit-run="
                (code, selectedLanguage) =>
                  onRunSubmitted(code, selectedLanguage)
              "
            ></omegaup-arena-runsubmit-popup>
          </template>
```

Reply:
Listo